docs/git-compatibility.md

```diff
@@ -153,6 +153,23 @@ repos may require you to deal with more involved Jujutsu and Git concepts.
   report any new ones you find, or if any of the known bugs are less minor than
   they appear.
 
+### Converting a repo into a co-located repo
+
+A Jujutsu repo backed by a Git repo has a full Git repo inside, so it is
+technically possible (though not officially supported) to convert it into a
+co-located repo like so:
+
+```bash
+# Move the Git repo
+mv .jj/repo/store/git .git
+# Tell jj where to find it
+echo -n '../../../.git' > .jj/repo/store/git_target
+# Unset core.bare that jj sets by default
+git config --unset core.bare
```

Comment:
Also add a line with `echo /.jj/ > .git/info/exclude`. I think we may also need something like this:

```
# Run an arbitrary `jj` command to cause it to set HEAD in the Git repo
jj st
```